Makefile

```diff
@@ -58,7 +58,10 @@ lock-python-ci-dependencies:
 package-protos:
 	cp -r ${ROOT_DIR}/protos ${ROOT_DIR}/sdk/python/feast/protos
 
-compile-protos-python:
+install-protoc-dependencies:
```

Comment:
Do we need this? Aren't they required and dev dependencies already anyway? I don't like this as we will end up mixing uv and pip in ci if we merge like this

Reply:
If users have already installed ci dependencies, it's of no use.